Sources/Server/Responder.swift

```diff
@@ -6,10 +6,19 @@ public class Responder {
 
     public init(){}
 
-    public func buildResponse(routeData: RouteData) -> String {
+    public func buildResponse(routeData: RouteData) -> Data {
         let responseLine = buildResponseLine(routeData: routeData)
         let headers = arrangeResponseHeaders(routeData: routeData)
-        return responseLine + headers + "\r\n\(routeData.body)"
+        let response = responseLine + headers + "\r\n\(routeData.body)"
+        return convertResponseToBytes(response: response, routeData: routeData)
+    }
+
+    private func convertResponseToBytes(response: String, routeData: RouteData) -> Data {
```

Comment:
Is there a reason the image needs to be it's own field on `routeData`? Since it is the 'body' of the response, do you think it make sense to share a field with that? 

Is this because you're reading the image in as data to begin with, but all other files get read in as strings?

Reply:
Yes, it was a simple fix given that the body was handled as a string. Now, that I know more about buffers I can have the body return Data as well. What do you think?